Samiksha/html/Assignment6/Assign6_1.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Question 1</title>
+        <link rel="stylesheet" href="Styles.css" />
+    </head>
+    <body>
+        <h1>MI-11 Intern HTML Assesment</h1>
+        <br />
+        <hr />
+        <h2><i>Personal Details</i></h2>
+        <form>
+            <label for="fname"><b>First Name:</b></label>
+            <input type="text" id="fname" name="fname" placeholder="First Name" /><br /><br />
```

Comment:
placeholders second words 1st letter should be small make this change in all input boxes